### ISSUE TITLE
Reproducing Global_Simple 

### ISSUE THREAD

**Issue by limorki** [vlgiitr/Group-Level-Emotion-Recognition#5]
It indeed a very clear and concise code that I was able to use it in Collaboratory very easily.
I am running the `DenseNet161_emotiW.py` to reproduce Global_Simple - finetune densnet161 on EmotiW 2018 .
I get better accuracy than the reported one in your paper - above the state of the art.
Paper's Table 1 : Val Acc  = 69.50%,  While running the code it outputs Best val Acc: 0.9359

```
Epoch 15/15
----------
0 Loss: 0.3900 Acc: 0.8424
1 Loss: 0.1979 Acc: 0.9359

Training complete in  482m 9.909503s
```
Are you aware of any bug that makes the result look too good?
What is your log output for `DenseNet161_emotiW.py`?

**Comment by dakshitagrawal97** [comment]
Hi there @limorki!

It seems like you trained the `FullDataset_DenseNet161_EmotiW.py` inside the `Models_FullTrained` folder.  The results mentioned in the paper are of models that are trained only on the Train Dataset, i.e. `DenseNet161_emotiW.py` inside the `Models_TrainDataset` folder.

However, I did retrain the `FullDataset_DenseNet161_EmotiW.py` model, just to check.  Turns out that you are getting a much better accuracy than ours.  The following is the log output:

```
Epoch 15/15
----------
0 Loss: 0.3801 Acc: 0.8486
1 Loss: 0.5872 Acc: 0.7730

Training complete in  301m 53.236140s
Best val Acc: 0.7760
```

I am not sure of any bug that makes the results look that good.  

Could you mention the environment in which you are training the model? I am using 4 Nvidia Tesla K80 in parallel for training.

**Comment by limorki** [comment]
Hi @dakshitagrawal97 
Thank you for putting the effort in running FullDataset_DenseNet161_EmotiW on your configuration, I really appreciate it.
I training on 1 gpu using Google's Collaboratory, this gist to reproduce the results:
[](https://colab.research.google.com/gist/limorki/84d83b7a43b523824ba642109ffac719/emotiw-train-no_face_densenet161.ipynb)
To run it you only need to modify project_root_dir = '' 

Can you please help me understand why you have used DenseNet161_emotiW.py and not the full dataset in your paper.

Thank you

**Comment by dakshitagrawal97** [comment]
Hi @limorki 

The train dataset and the validation dataset were defined by the emotiW dataset, provided to us by the organizers.  The `DenseNet161_emotiW.py` was trained according to that split.  

As a trick to increase the accuracy and for submitting to the challenge, we thought of training the model on the whole dataset as well.  But all the initial tests were done on the original split given in the emotiW dataset, hence we report those in the paper.
